src/app/layout/header/components/header-search/header-search.component.ts

```diff
@@ -25,27 +26,35 @@ export class HeaderSearchComponent implements OnInit {
   @Output()
   onSubCatClicked: EventEmitter<boolean> = new EventEmitter<boolean>();
   searchPlaceholder = environment.config.header.searchPlaceholder;
-
+  showGo = false;
   constructor(
     private store: Store<AppState>,
     private searchActions: SearchActions,
     private router: Router,
-    private activatedRouter: ActivatedRoute
+    private activatedRouter: ActivatedRoute,
+    private renderer: Renderer2,
```

Comment:
Remove `reneder2` & `ElementRef` from here. They seems to be unused?

Reply:
It is required

Reply:
Ok 👍